bigtable/bigtable.sh

```diff
@@ -109,14 +108,82 @@ EOF
     --clobber
 }
 
+function configure_bigtable_client_2x() {
+  cat <<EOF >hbase-site.xml
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+  <property><name>google.bigtable.project.id</name><value>${BIGTABLE_PROJECT}</value></property>
+  <property><name>google.bigtable.instance.id</name><value>${BIGTABLE_INSTANCE}</value></property>
+  <property>
+    <name>hbase.client.connection.impl</name>
+    <value>com.google.cloud.bigtable.hbase2_x.BigtableConnection</value>
+  </property>
+  <property>
+    <name>hbase.client.registry.impl</name>
+    <value>org.apache.hadoop.hbase.client.BigtableAsyncRegistry</value>
+  </property>
+  <property>
+    <name>hbase.client.async.connection.impl</name>
+    <value>org.apache.hadoop.hbase.client.BigtableAsyncConnection</value>
+  </property>
+  <!-- Spark-HBase-connector uses namespaces, which bigtable doesn't support. This has the
+  Bigtable client log warns rather than throw -->
+  <property><name>google.bigtable.namespace.warnings</name><value>true</value></property>
+</configuration>
+EOF
+
+  bdconfig merge_configurations \
+    --configuration_file "${HBASE_HOME}/conf/hbase-site.xml" \
+    --source_configuration_file hbase-site.xml \
+    --clobber
+}
+
+function configure_bigtable_client() {
+  local -r hbase_version=$(get_hbase_major_version)
+  #Update classpaths
+  cat <<'EOF' >>/etc/hadoop/conf/mapred-env.sh
+HADOOP_CLASSPATH="${HADOOP_CLASSPATH}:/usr/lib/hbase/*"
+HADOOP_CLASSPATH="${HADOOP_CLASSPATH}:/usr/lib/hbase/lib/*"
+HADOOP_CLASSPATH="${HADOOP_CLASSPATH}:/etc/hbase/conf"
+EOF
+
+  cat <<'EOF' >>/etc/spark/conf/spark-env.sh
+SPARK_DIST_CLASSPATH="${SPARK_DIST_CLASSPATH}:/usr/lib/hbase/*"
+SPARK_DIST_CLASSPATH="${SPARK_DIST_CLASSPATH}:/usr/lib/hbase/lib/*"
+SPARK_DIST_CLASSPATH="${SPARK_DIST_CLASSPATH}:/etc/hbase/conf"
+EOF
+
+  if [ "$hbase_version" == "2" ]
+  then
+    configure_bigtable_client_2x || err 'Failed to configure big table 2.x client.'
+  else
+    configure_bigtable_client_1x || err 'Failed to configure big table 1.x client.'
+  fi
+}
+
+function get_hbase_major_version() {
```

Comment:
We usually make conditions based on the Dataproc image version, it's available in `DATAPROC_VERSION` env variable.